projects/json-c/tokener_parse_ex_fuzzer.cc

```diff
@@ -0,0 +1,12 @@
+#include <stdint.h>
+
+#include <json.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const char *data, size_t size) {
```

Comment:
This signature is wrong; ubsan will fail. I'm going to modify this to take `unsigned char` and use a `reinterpret_cast`